src/FrontDoor/FrontDoor/ChangeLog.md

```diff
@@ -17,4 +17,8 @@
     * Overview of change #1
         - Additional information about change #1
 -->
-## Upcoming Release
\ No newline at end of file
+## Upcoming Release
+
+## Version 0.7.2
```

Comment:
@pichandwork you can remove this header and put all of the text under the `Upcoming Release` header